PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/AzurePlugin.java

```diff
@@ -101,39 +98,24 @@
 
     private String dataFile = PluginHelper.getTemplateFile(message("dataFileName"));
 
-    private final AzureSettings azureSettings;
+    private AzureSettings azureSettings;
 
     private String installationID;
 
     private Boolean firstInstallationByVersion;
 
-    public AzurePlugin(Project project) {
-        super(project);
+    @Override
+    public void runActivity(@NotNull Project project) {
```

Comment:
As far as I know, in UT mode, the project might be `null`

Reply:
This method is override from base class, please check the definition of the base class.
```java
public interface StartupActivity {
  ExtensionPointName<StartupActivity> POST_STARTUP_ACTIVITY = ExtensionPointName.create("com.intellij.postStartupActivity");

  /**
   * Please see https://github.com/JetBrains/intellij-community/blob/master/platform/service-container/overview.md#startup-activity
   */
  ExtensionPointName<StartupActivity.Background> BACKGROUND_POST_STARTUP_ACTIVITY = ExtensionPointName.create("com.intellij.backgroundPostStartupActivity");

  void runActivity(@NotNull Project project);

  interface DumbAware extends StartupActivity, com.intellij.openapi.project.DumbAware {
  }

  interface Background extends StartupActivity, com.intellij.openapi.project.DumbAware {}
}
```